.github/workflows/build_and_test.yml

```diff
@@ -872,6 +872,8 @@ jobs:
       run: |
         # We need this link because the jekyll build calls `python`.
         ln -s "$(which python3.9)" "/usr/local/bin/python"
```

Comment:
Can we remove the above two line from [SPARK-46935](https://issues.apache.org/jira/browse/SPARK-46935), @wayneguow ?

Reply:
Yes, good advice, this link is indeed introduced in this PR #44971 and only used in this place.